packages/app/src/components/pages/Home/ApplicationViewByStages.js

```diff
@@ -17,35 +22,65 @@ const ApplicationViewByStages = () => {
     return statuses.map( (status, index) => <option key={index} selected={status === currentlyViewing} value={status}>{status}</option> )
   }
 
+  const renderSortArrow = (columnName) => {
+    if (!sortConfig) {
+      return
+    }
+    return sortConfig.key === columnName ?
+      <FontAwesomeIcon icon={sortConfig.direction === 'ascending' ? "arrow-down" : "arrow-up"} />
+      : null
+  }
+
   return (
     <table className="new-files-table">
       <thead>
         <tr>
           <td className="add-padding-left new-files-title">
             <FontAwesomeIcon icon="file-image" color="green" />
-            <select onChange={e => setCurrentlyViewing(e.target.value)} className="stage-dropdown">
+            <select onChange={e => setCurrentlyViewing(e.currentTarget.value)} className="stage-dropdown">
               {renderOptionList()}
             </select>
           </td>
         </tr>
       </thead>
       <tbody>
         <tr className="home-new-files-headers">
-          <td className="add-padding-left"><h2><strong>Application Name</strong></h2></td>
-          <td><h2><strong>HP Name</strong></h2></td>
-          <td><h2><strong>Family Member Name</strong></h2></td>
-          <td><h2><strong>Received Date</strong></h2></td>
-          <td><h2><strong>Stage</strong></h2></td>
+          <td className="add-padding-left">
+            <h2 onClick={() => requestSort('nominationName')} style={{cursor: 'pointer'}}>
+              <strong>Application Name {renderSortArrow('nominationName')}</strong>
+            </h2>
```

Comment:
I think the sort arrow should be outside the `strong` and styled into the right position with CSS.
```suggestion
            <h2 onClick={() => requestSort('nominationName')} style={{cursor: 'pointer'}}>
              <strong>Application Name</strong>
              {renderSortArrow('nominationName')}
            </h2>
```